03-Raster-Calculations-In-R.Rmd

```diff
@@ -291,37 +303,33 @@ In the code we specify that the format will be GTiff. Overwrite is true so that
 it replaces any other file with this name. This is a useful setting so that you
 can make a change to the object and then update the file by re-runing the code.
 However, be careful to never accidentally re-use a name for a file that already 
-exists. We are setting the NA value to -9999, the National Ecological Observatory
-Network's (NEON) standard `NoData` value. 
-
-****
-
-> # Challenge: Explore the <a href="http://www.neoninc.org/science-design/field-sites/san-joaquin-experimental-range" target="_blank" >NEON San Joaquin (SJER) Field Site </a>
-
-> As ecologists we may want to explore the phenology of two different sites.  The 
-> <a href="http://www.neoninc.org/science-design/field-sites/san-joaquin-experimental-range" target="_blank" >NEON San Joaquin Experimental Range (SJER) field site </a> located in Southern California is very different ecosystem and climate compared to the 
->  <a href="http://www.neoninc.org/science-design/field-sites/harvard-forest" target="_blank" >NEON Harvard Forest Field Site</a> in 
-> Massachusetts.  
-
-> Import the San Joaquin DSM and DTM raster files and create a Canopy Height Model.
-> Then compare the two sites. BE sure to name your R objects and outputs carefully,
-> as follows: objectType_SJER (e.g. `DSM_SJER`). This will help you keep track of 
-> data from different sites!
->
-> 1. Import the DSM and DTM from the SJER directory. Don't forget to examine the
-data for CRS, bad values, etc. 
-> 2. Create a CHM from the two raster layers and check to make sure the data are 
-what you expect. 
-> 3. Plot the CHM from SJER. 
-> 4. Export as a GeoTIFF.
-> 5. What do you notice about tree heights at the two sites?  
-> Hint: plotting SJER and HARV data side-by-side would be an effective way to see the data at the same time.
->
->Compare your results with your neighbors. Do they look the same?
+exists. We are setting the NA value to -9999, the National Ecological
+Observatory Network's (NEON) standard `NoData` value. 
+
+# Challenge: Explore the NEON San Joaquin (SJER) Field Site
 
-****
+As ecologists we may want to explore the phenology of two different sites.  The 
+ <a href="http://www.neoninc.org/science-design/field-sites/san-joaquin-experimental-range" target="_blank" >NEON San Joaquin Experimental Range (SJER) field site </a> located in Southern California is very different ecosystem and climate compared to the 
+  <a href="http://www.neoninc.org/science-design/field-sites/harvard-forest" target="_blank" >NEON Harvard Forest Field Site</a> in 
+ Massachusetts.  
 
-```{r challenge-code-SJER-CHM, echo=FALSE}
+Import the San Joaquin DSM and DTM raster files and create a Canopy Height Model.
```

Comment:
let's be consistent with field site names throughout

SJER / San Joaquin Experimental Range